imports/ui/pages/flowdesigner.js

```diff
@@ -261,18 +265,18 @@ export class FlowDesigner extends React.Component {
                 </a>
               </li>
               <li className="tab col s2">
-                <a href="http://wireflow.co/mycharts">My charts</a>
+                <a href="/mycharts">My charts</a>
```

Comment:
What about using the [`Link` component](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md) `react-router` provides?

Reply:
Good thinking. I'll look into that.